Pod/Classes/MPOFaceServiceClient.m

```diff
@@ -248,24 +291,39 @@ - (NSURLSessionDataTask *)getPersonGroupWithPersonGroupId:(NSString *)personGrou
 - (NSURLSessionDataTask *)updatePersonGroupWithPersonGroupId:(NSString *)personGroupId name:(NSString *)name userData:(NSString *)userData completionBlock:(MPOCompletionBlock)completion {
     
     return [self startTaskWithHttpMethod:@"PATCH" path:[NSString stringWithFormat:@"persongroups/%@", personGroupId] parameters:@{@"name" : name, @"userData" : ObjectOrNull(userData)} urlParams:nil bodyData:nil completion:^(NSURLResponse *response, id responseObject, NSError *error) {
-       
+        
         [self runCompletionOnMainQueueWithBlock:completion error:error];
     }];
 }
 
 - (NSURLSessionDataTask *)deletePersonGroupWithPersonGroupId:(NSString *)personGroupId completionBlock:(MPOCompletionBlock)completion {
     
     return [self startTaskWithHttpMethod:@"DELETE" path:[NSString stringWithFormat:@"persongroups/%@", personGroupId] parameters:nil urlParams:nil bodyData:nil completion:^(NSURLResponse *response, id responseObject, NSError *error) {
-       
+        
         [self runCompletionOnMainQueueWithBlock:completion error:error];
     }];
 }
 
 - (NSURLSessionDataTask *)getPersonGroupsWithCompletion:(MPOPersonGroupArrayCompletionBlock)completion {
-    return [self startTaskWithHttpMethod:@"GET" path:@"persongroups" parameters:nil urlParams:nil bodyData:nil completion:^(NSURLResponse *response, id responseObject, NSError *error) {
+    
+    return [self listPersonGroupsWithStart:nil top:1000 completionBlock:completion];
+}
+
+- (NSURLSessionDataTask *)listPersonGroupsWithCompletion:(MPOPersonGroupArrayCompletionBlock)completion {
+    
+    return [self listPersonGroupsWithStart:nil top:1000 completionBlock:completion];
+}
+
+- (NSURLSessionDataTask *)listPersonGroupsWithStart:(NSString*)start top:(NSInteger)top completionBlock:(MPOPersonGroupArrayCompletionBlock)completion {
+    top = top > 1000 ? 1000 : top;
```

Comment:
delete this line, the parameter will be check by service and if bad parameter, a bad request will be returned